docs/ref/contrib/admin/index.txt

```diff
@@ -2886,6 +2886,24 @@ Templates can override or extend base admin templates as described in
     Raises ``django.contrib.admin.sites.NotRegistered`` if a model isn't
     already registered.
 
+.. method:: AdminSite.get_log_entries(request)
+
+    The new :meth:`AdminSite.get_log_entries()
+    <django.contrib.admin.AdminSite.get_log_entries>` returns a queryset
```

Comment:
This is the docs, not the release notes, so the "new" bit shouldn't be there. 

Can perhaps just begin, `Returns a ...`

Reply:
> Can perhaps just begin, Returns a ...

That's what I wrote in the first place, but @timgraham proposed otherwise.

Reply:
I think he was referring to the release note. 🤔

It's OK, thanks for updating. I will take another look this afternoon.